### PR TITLE
Create 04-everyday-shell.md

### PR DESCRIPTION
**Pull request by GrayR0ot**
Added a comprehensive Russian translation for the 'Everyday Shell' lesson, enhancing accessibility for Russian-speaking users and improving the project's global reach. Your contribution is greatly appreciated

### CODE REVIEW

**Comment by GrayR0ot** [comment]
@barraponto Request for review

**Comment by barraponto** [comment]
@GrayR0ot do you know any russian-speaking person who can come here and review the translation?
Izvinite, ya ne vladeju russkim...

File: translations/ru/04-everyday-shell.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+тема: по умолчанию
+Заголовок: Повседневная оболочка
+Информация: |
+  Использование  Git -- Урок  4: Повседневная оболочка
+   Подробнее на https://github.com/barraponto/usegit
+переход: слайд влево
+рисунки:
+  сохранять: false
+имяФайлаДляЭкспорта: 04-повседневная-оболочка.pdf
+---
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:11]: Also, I'm pretty sure the keywords must stay in english for slidev to correctly interpret them.
The values can be in Russian, though.

**Review by barraponto — CHANGES_REQUESTED** [review]

File: translations/ru/04-everyday-shell.md (hunk 1 above)
Comment (amniriane) [reply translations/ru/04-everyday-shell.md:11]: it's ok

Comment (barraponto) [reply translations/ru/04-everyday-shell.md:11]: I mean, in lines like `Заголовок: Повседневная оболочка`, `Заголовок` must be `title` for the software to interpret it correctly. The value can stay in Russian, although some values are actually identifiers, like `default`, `slide-left` and `statement`

**Review by barraponto — CHANGES_REQUESTED** [review]

**Comment by GrayR0ot** [comment]
fixed header values matching source

File: translations/ru/04-everyday-shell.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+theme: по умолчанию
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:2]:
```suggestion
theme: default
```

File: translations/ru/04-everyday-shell.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+# Повседневная оболочка
+
+## c Capi Etheriel
+
+[@barraponto](https://github.com/barraponto)
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:20]:
```suggestion
layout: intro
```

File: translations/ru/04-everyday-shell.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+# Поиск файлов
+
+- tldr ls
+- tldr find
+- tldr tree
+
+<!-- show recipe examples and code examples too. -->
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:32]:
```suggestion
layout: intro
```

File: translations/ru/04-everyday-shell.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+# Поиск в файлах
+
+- tldr grep
+- tldr rg
+
+<!-- show recipe examples and code examples too. -->
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:43]:
```suggestion
layout: intro
```

File: translations/ru/04-everyday-shell.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+# Потоки
+
+- standard input (stdin)
+- standard output (stdout)
+- standard error (stderr)
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:53]:
```suggestion
layout: intro
```

File: translations/ru/04-everyday-shell.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+# Перенаправление потока
+
+- the pipe |
+- the output >
+- the append >>
+- the input <
+- the tee
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:65]:
```suggestion
layout: intro
```

File: translations/ru/04-everyday-shell.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+# 
+# Композиция
+
+- how many recipes with eggs?
+
+---
+макет: вступление
```
Comment (barraponto) [inline translations/ru/04-everyday-shell.md:74]:
```suggestion
layout: intro
```

**Review by barraponto — CHANGES_REQUESTED** [review]

**Pull request closed without merging**